File: Php/poo/figure/library/rectangle.php
<?php

    class rectangle extends figureGeometric{
    
        private $base;
        private $height;
        private $calculate;
    
    public function __construct($base,$height){
        parent::__construct("Rectangle");
        $this->base = $base;
        $this->height = $height;
    }

    public function getBase(){
        return $this->base;
    }

    public function getHeight(){
        return $this->height;
    }

    public function calculateArea(): float {
        $this->calculate = ($this->getBase() * $this->getHeight());
        return $this->calculate;
    }
}

File: Java/poo/figure/library/rectangle.java
package library;

public class rectangle extends figureGeometric {
    private double base;
    private double height;

    public rectangle(double base, double height) {
        super("Rectángulo");
        this.base = base;
        this.height = height;
    }

    public double getBase() {
        return this.base;
    }

    public double getHeight() {
        return this.height;
    }

    @Override
    public double calculateArea() {
        return this.base * this.height;
    }
}
